### DISCUSSION TITLE
Validator sanitizor in v5 ?

### DISCUSSION THREAD

**Discussion by popovicieduard** [adonisjs/core#1368] [Help] [answered]
Is there an equivalent in v5 ?
```
sanitizationRules() {
    return {
      currency: 'toUpper',
    }
}
```

provider:
```
const Validator = use('Validator');
Validator.sanitizor.toUpper = value => String(value).toUpperCase()
```

**Comment by thetutlage** [comment, marked as answer]
Nope. We have remove sanitization layer in v5 and merged the required rules within the validation rules. So everything stays together in one place and multiple loops are not required.

I believe, case management can also be added to the `string` rule. I can accept the PR, if someone wants to create it or atleast create it as an issue in the Validator repo, so I can get back to it
